benchmarks/scale_1k_20k/ct_scale_1k_20k.yaml

```diff
@@ -0,0 +1,22 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 9999999
+
+head_node_type:
+    name: head_node
+    instance_type: r5dn.16xlarge # Network optimized.
+    resources:
+      cpu: 0
+      custom_resources:
+        node: 1
+
+worker_node_types:
+    - name: small_worker
+      instance_type: m5.xlarge
+      min_workers: 1000
+      max_workers: 9999999
```

Comment:
Is this correct? looks like it will scale more than 1K nodes in this case? (our global max workers is also large)

Reply:
yes, correct. we scale it to 1k nodes in script right now.